java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -28,5 +28,5 @@ abstract class AbstractArcHost : ArcHost {
         particles.remove(particle)
     }
 
-    override fun registeredParticles(): List<KClass<out Particle>> = particles
+    override val registeredParticles: List<KClass<out Particle>> get() = particles
```

Comment:
Nit: not sure if ktlint will complain about this, but the getter should go on the next line.

https://developer.android.com/kotlin/style-guide#properties

Reply:
Not done?